word_occupancy/src/main.cpp

```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+#include <unordered_map>
+#include "print.h"
+#include "map_create.h"
+
+int main()
+{
+    std::unordered_map<std::string, int> umap;
+    std::string input;
+
+    getline(std::cin, input);
```

Comment:
Upgrade your program. Now your program must get a file path and read the string from there

Reply:
+

Reply:
File path must be passed as program argument, not STDIN